src/transformers/models/t5/modeling_t5.py

```diff
@@ -2019,7 +2020,9 @@ def forward(
         >>> print(tokenizer.decode(outputs[0], skip_special_tokens=True))
         >>> # studies have shown that owning a dog is good for you.
         ```"""
-
+        # temp fix
+        use_cache = False
```

Comment:
do you want it to stay like this ?

Reply:
Yes, until we will get back and fix this issue or if Michal says it takes too long to generate :)